src/utils/ValidationHelper.ts

```
@@ -182,6 +182,16 @@ export class ValidationHelper {
 		}
 	}
 
+	// If >1 addresses in shared_claims, address{}.validUntil will not be present on the preferredAddress
+	getPreferredAddress(sharedClaimsAddress: PersonIdentityAddress[]): PersonIdentityAddress | undefined {
+		if (sharedClaimsAddress.length > 1) {
```

If less than one should we throw an error. Is a journey without any address valid

---

We validate the presence of address and specific fields directly before this code is run in the session lambda

---

But in the event there is no addresses should we not fail here?